dialogflow-read.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+function Read() {
```

Comment:
Usually read functions are expecting a return value. I noticed that you put the results in console, better to change it. Maybe use callback function, or async syntax.